scripts/build-shotcut.sh

```diff
@@ -860,6 +875,12 @@ function set_globals {
   fi
   CFLAGS_[17]="$CFLAGS"
   LDFLAGS_[17]=$LDFLAGS
+
+  #####
+  # rubberband
+  CONFIG[18]="./configure --prefix=$FINAL_INSTALL_DIR --disable-programs --disable-vamp --disable-ladspa"
```

Comment:
To fix Windows build I added
```
  if test "$TARGET_OS" = "Win32" -o "$TARGET_OS" = "Win64" ; then
    CONFIG[18]="${CONFIG[18]} --host=$HOST"
  fi
```

Reply:
For the link errors, do we link statically on Windows? If so, does it work to add this to rubberband.pc:

```
Libs.private: -lfftw3 -lsamplerate
```

If that works, I can submit a patch.

Reply:
I have fixed it locally. I will patch it when merging. On Windows, we have to use `-lfftw3-3` which I think is unusual and should not be sent upstream. My patch is like `sed 's/-lrubberband/-lrubberband -lfftw3-3 -lsamplerate/' -i rubberband.pc.in`. I would have to run another test using Libs.private.

Reply:
Ok. Understood. If you do come up with any patches that can be sent upstream, let me know and I would be happy to submit them.

Reply:
I noticed this:
https://github.com/breakfastquay/rubberband/blob/master/configure#L1344
LIBS        libraries to pass to the linker, e.g. -l<library>
FFTW_LIBS   linker flags for FFTW, overriding pkg-config

So maybe you can add: 
FFTW_LIBS=-lfftw3-3
or
LIBS="-lfftw3-3 -lsamplerate"

Or, if you are happy with the sed work-around, maybe it is fine.

Reply:
To clarify, rubberband builds, but it is the MLT module failing to link. `Libs.private: -lfftw3-3 -lsamplerate` does not work because `pkg-config --libs rubberband` does not emit them, and we are linking statically to rubberband because for some reason it is build .so files under under cross-compile in addition to the static link library. So, the MLT builds only accepts the static lib. I am not particularly to keen to debug that build issue in their project.

` FFTW_LIBS` is used to override what is in fftw3.pc, but that is correct in our win32 prebuild bundle. I doubt that will help the MLT build. I will stick to this sed hack for now.